modules/fbx_importer/data/fbx_mesh_data.cpp

```diff
@@ -539,8 +513,8 @@ HashMap<int, T> FBXMeshData::extract_per_vertex_data(
 				for (size_t polygon_vertex_index = 0; polygon_vertex_index < p_fbx_data.index.size(); polygon_vertex_index += 1) {
 					const int vertex_index = get_vertex_from_polygon_vertex(p_polygon_indices, polygon_vertex_index);
 					ERR_FAIL_COND_V_MSG(vertex_index < 0, (HashMap<int, T>()), "FBX file corrupted: #ERR8");
-					ERR_FAIL_COND_V_MSG(vertex_index >= p_vertex_count, (HashMap<int, T>()), "FBX file seems corrupted: #ERR9.")
-					ERR_FAIL_COND_V_MSG(p_fbx_data.index[polygon_vertex_index] < 0, (HashMap<int, T>()), "FBX file seems corrupted: #ERR10.")
+					ERR_FAIL_COND_V_MSG(vertex_index >= p_vertex_count, (HashMap<int, T>()), "FBX fileseems  corrupted: #ERR9.")
```

Comment:
Typo in message.

fileseems -> file seems

Reply:
Err 9 is just a way for us to know the kind of error, in case someone reports the issue.
In case of the file is corrupted, or seems so, there is no good way to tell what's going on to the user, nor is possible for him fix that. So the best message is "The file is corrupted".

So it's much better just say that, and return a number so in case of issue report, we know where the corruption is.